src/main/java/org/jboss/ejb/client/EJBClientContext.java

```diff
@@ -337,6 +349,18 @@ public boolean registerEJBReceiver(final EJBReceiver receiver) {
         return this.registerEJBReceiver(receiver, null);
     }
 
+    public boolean registerEJBReceiver(final EJBReceiver receiver, boolean deleteExistingReceiver) {
```

Comment:
In reality, we are not : https://github.com/jbossas/jboss-ejb-client/pull/201/files#diff-4666ce813681f663f6e65e86f9cc1028R348